view/base.js

```diff
@@ -338,8 +341,8 @@ if ( typeof kintInitialized === 'undefined' ) {
 			, shiftKey = e.shiftKey
 			, i = kint.currentPlus;
 
-
-		if ( keyCode === 68 ) { // 'd' : toggles navigation on/off
+		if ( keyCode === 68 || (keyCode === 74 && j_toggles) ) { // 'd' or 'j' : toggles navigation on/off
```

Comment:
So 'j' toggles navigation as well as navigate - but how do you get out of kint navigation mode and just want to scroll the page with arrow keys? 'd'? That's a bit counter-intuitive, don't you think? I'm not a VIM buff, so what's your oppinion?